### ISSUE TITLE
Auto-deploy only new content

### ISSUE THREAD

**Issue by meganhkelley** [department-of-veterans-affairs/va.gov-team#18847]
## Details 

Dependent on #18844
Could be done in parallel with #18845 with close coordination

We don't want to waste time deploying if there are no changes in content. This ticket involves archiving the CMS Export data that we get from Drupal when we deploy successfully through the pipeline. Then, on subsequent deploys, compare the newly downloaded CMS Export data with the previously archived data to confirm that there are changes. 

If no changes, pipeline stops. 
If changes, pipeline continues. 

Suggested comparison involves using hashes or md5 checksums of the downloaded data.

Additional details in design doc: https://github.com/department-of-veterans-affairs/va.gov-team/pull/17304/files

## Validation Options

* Complete this ticket after #18845 (job runs hourly), and review logs after 1-2 days to confirm that there were instances where the pipeline stopped when no changes were found
* Manually kick off deploy in quick succession (with or without hourly scheduled deploy), to confirm that pipeline stops when no content changes are found

## Acceptance Criteria

- [ ] Determine to how to validate that this is working (options above)
- [ ] Validate that content-only deploy pipeline stops when no content changes have been made
- [ ] CMS data is archived to the right place in S3

**Comment by meganhkelley** [comment]
This implementation is specific to CMS Export, which is no longer relevant. 

Closing. If we pursue a GraphQL solution for this we will spin up a new ticket for it.
